src/server/graphql/queries/index.js

```diff
@@ -1,5 +1,41 @@
-import autoloader from 'auto-loader'
+import findChapters from 'src/server/graphql/queries/findChapters'
+import findMembers from 'src/server/graphql/queries/findMembers'
+import findPhaseSummaries from 'src/server/graphql/queries/findPhaseSummaries'
+import findPhases from 'src/server/graphql/queries/findPhases'
+import findProjects from 'src/server/graphql/queries/findProjects'
+import findProjectsForCycle from 'src/server/graphql/queries/findProjectsForCycle'
+import findRetrospectiveSurveys from 'src/server/graphql/queries/findRetrospectiveSurveys'
+import findUsers from 'src/server/graphql/queries/findUsers'
+import getChapter from 'src/server/graphql/queries/getChapter'
+import getCycleById from 'src/server/graphql/queries/getCycleById'
+import getCycleVotingResults from 'src/server/graphql/queries/getCycleVotingResults'
+import getMemberById from 'src/server/graphql/queries/getMemberById'
+import getProject from 'src/server/graphql/queries/getProject'
+import getProjectSummary from 'src/server/graphql/queries/getProjectSummary'
+import getProjectSummaryForMember from 'src/server/graphql/queries/getProjectSummaryForMember'
+import getRetrospectiveSurvey from 'src/server/graphql/queries/getRetrospectiveSurvey'
+import getRetrospectiveSurveyQuestion from 'src/server/graphql/queries/getRetrospectiveSurveyQuestion'
+import getUser from 'src/server/graphql/queries/getUser'
+import getUserSummary from 'src/server/graphql/queries/getUserSummary'
 
-import {pruneAutoLoad} from 'src/server/graphql/util'
-
-export default pruneAutoLoad(autoloader.load(__dirname))
+export default {
+  findChapters,
+  findMembers,
+  findPhaseSummaries,
+  findPhases,
+  findProjects,
+  findProjectsForCycle,
+  findRetrospectiveSurveys,
+  findUsers,
+  getChapter,
+  getCycleById,
+  getCycleVotingResults,
+  getMemberById,
+  getProject,
+  getProjectSummary,
+  getProjectSummaryForMember,
+  getRetrospectiveSurvey,
+  getRetrospectiveSurveyQuestion,
+  getUser,
+  getUserSummary,
```

Comment:
Protip: can accomplish this in half the lines:

```js
export {default as findChapters} './findChapters'
export {default as findMembers} './findMembers'
export {default as findPhaseSummaries} './findPhaseSummaries'
export {default as findPhases} './findPhases'
export {default as findProjects} './findProjects'
export {default as findProjectsForCycle} './findProjectsForCycle'
export {default as findRetrospectiveSurveys} './findRetrospectiveSurveys'
export {default as findUsers} './findUsers'
export {default as getChapter} './getChapter'
export {default as getCycleById} './getCycleById'
export {default as getCycleVotingResults} './getCycleVotingResults'
export {default as getMemberById} './getMemberById'
export {default as getProject} './getProject'
export {default as getProjectSummary} './getProjectSummary'
export {default as getProjectSummaryForMember} './getProjectSummaryForMember'
export {default as getRetrospectiveSurvey} './getRetrospectiveSurvey'
export {default as getRetrospectiveSurveyQuestion} './getRetrospectiveSurveyQuestion'
export {default as getUser} './getUser'
export {default as getUserSummary} './getUserSummary'
```

Reply:
Making this switch leads to 49 failing tests on my end. Have tried a variety of ways of exporting and have not found a workable solution.

Reply:
Got it. I'll have a look.